src/Web/WebClient.php

```diff
@@ -53,81 +53,108 @@ class WebClient
 	const ANDROIDTABLET = 22;
 	const EDGE          = 23;
 	const BLINK         = 24;
+	const EDG           = 75;
 
 	/**
-	 * @var    integer  The detected platform on which the web client runs.
+	 * The detected platform on which the web client runs.
+	 *
+	 * @var    integer
 	 * @since  1.0
 	 */
 	protected $platform;
 
 	/**
-	 * @var    boolean  True if the web client is a mobile device.
+	 * True if the web client is a mobile device.
+	 *
+	 * @var    boolean
 	 * @since  1.0
 	 */
 	protected $mobile = false;
 
 	/**
-	 * @var    integer  The detected rendering engine used by the web client.
+	 * The detected rendering engine used by the web client.
+	 *
+	 * @var    integer
 	 * @since  1.0
 	 */
 	protected $engine;
 
 	/**
-	 * @var    integer  The detected browser used by the web client.
+	 * The detected browser used by the web client.
+	 *
+	 * @var    integer
 	 * @since  1.0
 	 */
 	protected $browser;
 
 	/**
-	 * @var    string  The detected browser version used by the web client.
+	 * The detected browser version used by the web client.
+	 *
+	 * @var    string
 	 * @since  1.0
 	 */
 	protected $browserVersion;
 
 	/**
-	 * @var    array  The priority order detected accepted languages for the client.
+	 * The priority order detected accepted languages for the client.
+	 *
+	 * @var    array
 	 * @since  1.0
 	 */
-	protected $languages = array();
+	protected $languages = [];
```

Comment:
For master, long array syntax needs to stay in use (PHP 5.3 support).

Reply:
Which branch do you prefer for new pr's? i just did copy paste frome the orginal J4.0 branch.

Reply:
J4 is using the 2.0-dev branch

J3 is using the master branch (to a lesser extent but still in use)

Generally, if it's B/C it's safe to land in master would be my approach to things, unless you're PR'ing a change specifically made in the 2.0 branch.

Reply:
(Same statement applies for all the Framework repos the CMS is pulling in BTW as it relates to branches)